ui/packages/ui/src/app/Layout/appHeader.tsx

```diff
@@ -18,10 +23,33 @@ const AppHeader: React.FC<IAppHeader> = (props) => {
 	const homeClick = () => {
 		history.push("/")
 	}
-
+	const [isModalOpen, setIsModalOpen] = React.useState(false);
+	const handleModalToggle = () => setIsModalOpen(!isModalOpen);
+	const commitLink = `https://github.com/debezium/debezium-ui/commit/${process.env.COMMIT_HASH}`;
+	const Footer = () => (
+		<p>Build <a href={commitLink} target="_blank">{process.env.COMMIT_HASH}</a></p>
+	);
+	const BuildModal = () => (
```

Comment:
I was tempted to put this in it's own file but wasn't sure what the preferences are for that.

Reply:
I think it's fine for now.  We can move it later if we start adding more info.